src/wallet/wallet.h

```diff
@@ -771,6 +773,12 @@ class CWallet final : public WalletStorage, public interfaces::Chain::Notificati
       * output itself, just drop it to fees. */
     CFeeRate m_discard_rate{DEFAULT_DISCARD_FEE};
 
+    /** When the actual feerate is less than the consolidate feerate, we will tend to make transactions which
+     * consolidate inputs. When the actual feerate is greater than the consolidate feerate, we will tend to make
+     * transactions which have the lowest fees.
+     */
+    CFeeRate m_consolidate_feerate{DEFAULT_CONSOLIDATE_FEERATE};
```

Comment:
we don't really need any of this; at least we should set DEFAULT_CONSOLIDATE_FEERATE to 0 or something to basically disable this logic?

Reply:
Not sure if this is the best idea since `m_consolidate_feerate` is used to define `coin_selection_params.m_long_term_feerate` in `CreateTransactionInternal` ([source](https://github.com/dashpay/dash/commit/937fcc9b18c4c64ef8978636e0c1c20dd81b5d9a#diff-6e06b309cd494ef5da4e78aa0929a980767edd12342137f268b9219167064d13R638-L687)) though I did find that the old code generated a fee-per-KB of `1000` and the commit changes it to a fee-per-KB of `10000`.

This has been corrected in latest push.